doc/configuration.rst

```diff
@@ -749,52 +736,42 @@ user
 Configuration at Task Level
 ---------------------------
 
-Luigi also supports to define some configurations at task level like scheduling configurations. It is defined in ``config`` variable in the task.
+Luigi also supports to define some configurations at task level like scheduling configurations. It is as their names as a property variable in the task.
 
 .. code-block:: python
 
     class GenerateWordsFromHdfs(luigi.Task):
 
-        config = {
-          'disable-num-failures':5
-        }
+       disable_num_failures = 5
 
         ...
 
-    class GenerateWordsFromRDB(luigi.Task):
+    class GenerateWordsFromRDBM(luigi.Task):
 
-        config = {
-          'disable-num-failures':3
-        }
+       disable_num_failures = 3
 
         ...
 
     class CountLetters(luigi.Task):
 
-        config = {
-          'upstream-status-when-all':True
-        }
-
         def requires(self):
-            return [GenerateWordsFromHdfs(),GenerateWordsFromRDB()]
+            return [GenerateWordsFromHdfs(),GenerateWordsFromRDBM()]
 
         ...
 
 Supported Configurations
 ************************
 The configrations below are also definable in luigi config file. Check :ref:`scheduler-config`
 
-+--------------------------+-----------+
-| Config                   | Section   |
-+==========================+===========+
-| disable-num-failures     | scheduler |
-+--------------------------+-----------+
-| disable-hard-timeout     | scheduler |
-+--------------------------+-----------+
-| disable-window-seconds   | scheduler |
-+--------------------------+-----------+
-| upstream-status-when-all | scheduler |
-+--------------------------+-----------+
++------------------------+-----------+
+| Config                 | Section   |
++========================+===========+
+| retry-count            | scheduler |
++------------------------+-----------+
+| disable-hard-timeout   | scheduler |
```

Comment:
The correct in config is to always use underscores. There are places where that's not the case, those cases are just yet not updated. :)